### DISCUSSION TITLE
[Enhacement] Better organization with a better usage of GitHub Projects board

### DISCUSSION THREAD

**Discussion by Cfun1** [xamarin/XamarinCommunityToolkit#726] [Ideas]
### Description
Moved from https://github.com/xamarin/XamarinCommunityToolkit/issues/722
I am just opening this here to draw some attention and possibly avoid things get messy. By Keeping a proper tracking of issues with appropriate Labels and projects assignment, we will have a better overview of what is going on and where are we.

A reference would be [Xamarin.forms repo projects](https://github.com/xamarin/Xamarin.Forms/projects).
 Any ideas are very welcome to fill a concrete task list in the goal to have a better organization.

Some labels have a "prefix" "a/", "p/" "s/" any particular meaning of those ?

Task Item suggestion:

- [ ] vCurrent, vNext not always used. 
- [ ] Updating existing projects with old issues that are unassigned to any project. (CameraView and MediaElement  Expander project that are not update since a while.)
- [ ] Configuring Project automation for new issues (add automatically to New/Triage for example).
- [ ] Setup project automation for new issues to be moved to project Triage/New.
- [ ] In-progress label, and linking PR to issues (for develop branch, [GitHub closes..key words work only on default branch](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)) not always updated early.

-  Revision:
      - [x] Go-through old feature request that still having the [needs-approval](https://github.com/xamarin/XamarinCommunityToolkit/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-approval) label. 
    - [ ]  Go-through the [Unverified Bugs](https://github.com/xamarin/XamarinCommunityToolkit/issues?q=is%3Aissue+is%3Aopen+label%3As%2Funverified+%2B+bug) list.

**Comment by jfversluis** [comment]
> Some labels have a "prefix" "a/", "p/" "s/" any particular meaning of those ?

This is also how Forms does it :) 
* **a/** == area i.e. area/MediaElement
* **p/** == platform i.e. platform/iOS
* **s/** == status i.e. status/unverified

Very open to using board more in a good way. What would be a way to set this up for "outsiders", with outsiders I mean people in the wild, a contributor visiting our repo for the first time. What would they look for? How can we make clear what issues are there for them to pick up? 

Or how can we make clear to users what the status of a certain issue is? Do you have any concrete ideas on how we can arrange things to make that better?

**Reply by VladislavAntonyuk** [reply]
If we are talking about the board I suggest this columns:
- Backlog - new issues added here
- Defined - items which are ready for implementation. 
- in progress - I think it is clear 
- in review - optional but would be easier to track
- done

**Reply by pictos** [reply]
@VladislavAntonyuk for the board, we use the `Backlog` as `Triage` and `Defined` as `Ready for work/implementation`. For now, we put issues on the board manually, so you may see some delay.

**Comment by Cfun1** [comment]
@jfversluis @pictos any update on "Setup project automation for new issues to be moved to project Triage/New" ?

**Reply by pictos** [reply]
I don't know how to do that, normally I move items on the board manually, but these days I would not have time to do it. So I'm just trying to update the labels and code-reviews
